### PR TITLE
Browser support changes

### LINKED ISSUE #2695: Browser support
We will change our browser support polity to be the following:

- Chrome desktop (any Blink-based, including Android WebView, Opera and new Edge): two latest releases
- Firefox desktop: two latests releases + ESR
- Safari desktop: two latest releases
- iOS: three latest major releases

We already list those as supported browser but now we define which versions are supported and dropping IE.

With this we can drop all polyfills we currently use. We still need systemjs for module loading on the worker side.

### PR DESCRIPTION
**Pull request by charlag**
close #2695

Drop polyfills, drop some client detection, better start error message

### CODE REVIEW

**Comment by charlag** [comment]
Needs better handling for WebViews on Android, otherwise should be ready

**Comment by charlag** [comment]
Ready for review

File: buildSrc/index.template.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	const div = document.createElement("div")
+	div.style.fontFamily = "sans-serif"
+	div.style.textAlign = "center"
+	div.style.fontSize = "24px"
+	document.body.appendChild(div)
+
+	const text = document.createElement("p")
+	const isAndroidApp = navigator.userAgent.indexOf("Android") !== -1 && navigator.userAgent.indexOf("wv") !== -1
+	if (isAndroidApp) {
+		text.textContent = "Seems like your system WebView is outdated. Please see FAQ entry for more information"
```
Comment (johnbotris) [inline buildSrc/index.template.js:26]: Can we do translations here?

File: dist.js [hunk 2]
```diff
@@ -38,7 +37,8 @@ const __dirname = dirname(fileURLToPath(import.meta.url))
 // which looks like a problem e.g. for accessing fields.
 
 // change this to disable minification and name mangling.
-const MINIFY = true
+// FIXME
```
Comment (johnbotris) [inline dist.js:40]: still need to revert this

**Pull request closed without merging**